### ISSUE TITLE
The impacts of BatchNorm instead of LayerNorm

### ISSUE THREAD

**Issue by Ye-D** [DachengLi1/MPCFormer#1]
In the CrypTen-based MPCFormer, such as MPC-Bert, I noticed that the  LayerNorm is replaced by BatchNorm (due to CrypTen does not support LayerNorm now). Will this modification influence Bert's performance (e.g., accuracy)? Due to the source code did not give the model-loading scripts, it might be difficult for me to check it in experiments. Thanks for your help.

**Comment by DachengLi1** [comment]
@Ye-D thanks for the great question! We measure accuracy in plain PyTorch, and only system throughput with Crypten, as reported in the paper. We are implementing modules such as Layernorm in Crypten to measure the accuracy. 

By module loading, do you mean something like [this](https://github.com/DachengLi1/MPCFormer/blob/main/src/benchmark/profile_transformer.py#L56) ?

**Comment by Ye-D** [comment]
@DachengLi1 Thanks for the reply!
